tests/PromiseTest/PromiseFulfilledTestTrait.php

```diff
@@ -191,7 +191,7 @@ public function thenShouldContinueToExecuteCallbacksWhenPriorCallbackSuspendsFib
         $adapter->resolve(42);
 
         $fiber = new \Fiber(function () use ($adapter) {
-            $adapter->promise()->then(function (int $value) {
+            $adapter->promise()->then(function (int $value) { /** @phpstan-ignore-line */
```

Comment:
Is this still needed?

Reply:
Removing that yields the following error:

```
Parameter #1 $onFulfilled of method React\Promise\PromiseInterface<mixed>::then() expects (callable(mixed): void)|null, Closure(int): void given.  
         💡 Type int of parameter #1 $value of passed callable needs to be same or wider than parameter type mixed of accepting callable. 
```

Can try and find a way to make it work, but this is one of those where I rather be lazy and ignore it to focus on the more important parts of the PR.

Reply:
Resolved :shipit: